client/src/App.css

```diff
@@ -52,3 +52,13 @@
 .namesContainer {
   margin-bottom: 15px;
 }
+
+/* Disable hover on mobile screen sizes */
+@media only screen and (max-width: 768px) {
+  /* Chakra UI default className for the bid and actual buttons */
+  .css-4o5e5l:hover, 
```

Comment:
ChakraUI has its own way of overriding default functionality. Please follow this documentation:

https://chakra-ui.com/docs/styled-system/customize-theme

Reply:
TY for docs was looking for this.